modules/jvm/api/module.yaml

```diff
@@ -21,6 +21,9 @@ envs:
 - name: JAVA_MAX_INITIAL_MEM
   description: Is used when no `-Xms` option is given in **JAVA_OPTS**. This is used to calculate the maximum value of the initial heap memory. If used in a container without any memory constraints for the container then this option has no effect. If there is a memory constraint then `-Xms` is limited to the value set here. The default is 4096MB which means the calculated value of `-Xms` never will be greater than 4096MB. The value of this variable is expressed in MB.
   example: "4096"
+- name: MAX_RAM_PERCENTAGE
+  description: Set this to alter the value for the `-XX:MaxRAMPercentage` parameter from the default value of 80, corresponding to 80%.
+  example: "100"
```

Comment:
An example of `100` seems dubious. It suggests setting the heap to `100%` makes sense leaving no room for non-heap memory like thread stacks, code cache, metaspace, etc. Perhaps `example: 90`?